packages/tools/generate-images.ts

```diff
@@ -91,475 +111,396 @@ function sourceById(id: number) {
-		width: 4820,
-		height: 2938,
-	},
-	{
-		source: 9,
-		dest: 'Assets/WebsiteAssets/images/home-top-img-cn-2x.png',
-		width: 2388,
-		height: 1456,
-	},
-	{
-		source: 9,
-		dest: 'Assets/WebsiteAssets/images/home-top-img-cn-2x.webp',
-		width: 2388,
-		height: 1456,
-	},
-	{
-		source: 9,
-		dest: 'Assets/WebsiteAssets/images/home-top-img-cn.png',
-		width: 1205,
-		height: 734,
-	},
-	{
-		source: 9,
-		dest: 'Assets/WebsiteAssets/images/home-top-img-cn.webp',
-		width: 1205,
-		height: 734,
-	},
-
-	// ============================================================================
-	// Joplin Server Icons
-	// ============================================================================
-
-	{
-		source: 10,
-		dest: 'packages/server/public/images/icons/server/icon-512.png',
-		width: 512,
-		height: 512,
-	},
-	{
-		source: 10,
-		dest: 'packages/server/public/images/icons/server/icon-192.png',
-		width: 192,
-		height: 192,
-	},
-	{
-		source: 10,
-		dest: 'packages/server/public/images/icons/server/icon-180.png',
-		width: 180,
-		height: 180,
-	},
-	{
-		source: 10,
-		dest: 'packages/server/public/images/server_logo.png',
-		width: 512,
-		height: 512,
-	},
-	{
-		source: 10,
-		dest: 'packages/server/public/images/icons/server/icon-32.png',
-		width: 32,
-		height: 32,
-		imageName: 'joplinServer32',
-	},
-	{
-		source: 10,
-		dest: 'packages/server/public/images/icons/server/icon.svg',
-	},
-	{
-		source: 10,
-		dest: 'packages/server/public/images/icons/server/favicon.ico',
-		images: ['joplinServer32'],
-	},
-
-	// ============================================================================
-	// Joplin Cloud Icons
-	// ============================================================================
+const readAppIcon = async (filePath: string): Promise<AppIconContents> => {
+	if (!(await pathExists(filePath))) return { images: [] };
+	const content = await readFile(filePath, 'utf8');
+	return JSON.parse(content) as AppIconContents;
+};
 
-	{
-		source: 11,
-		dest: 'packages/server/public/images/icons/cloud/icon-512.png',
-		width: 512,
-		height: 512,
-	},
-	{
-		source: 11,
-		dest: 'packages/server/public/images/icons/cloud/icon-192.png',
-		width: 192,
-		height: 192,
-	},
-	{
-		source: 11,
-		dest: 'packages/server/public/images/icons/cloud/icon-180.png',
-		width: 180,
-		height: 180,
-	},
-	{
-		source: 11,
-		dest: 'packages/server/public/images/cloud_logo.png',
-		width: 512,
-		height: 512,
-	},
-	{
-		source: 12,
-		dest: 'packages/server/public/images/icons/cloud/icon-32.png',
-		width: 32,
-		height: 32,
-		imageName: 'joplinCloud32',
-	},
-	{
-		source: 12,
-		dest: 'packages/server/public/images/icons/cloud/icon.svg',
-	},
-	{
-		source: 12,
-		dest: 'packages/server/public/images/icons/cloud/favicon.ico',
-		images: ['joplinCloud32'],
-	},
-];
+const getOperations = async (rootDir: string): Promise<Operation[]> => {
+	const AppIconSet = 'packages/app-mobile/ios/Joplin/Images.xcassets/AppIcon.appiconset';
```

Comment:
```suggestion
	const appIconSet = 'packages/app-mobile/ios/Joplin/Images.xcassets/AppIcon.appiconset';
```

We use camelCase for constants too